test/event_test.dart

```diff
@@ -1200,6 +1212,581 @@ void main() {
+        editHtml: false,
+      );
+      testUnlocalizedBody(
+        expectation: 'body',
+        plaintextBody: false,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: true,
+        isEdit: true,
+        editBody: null,
+        editFormattedBody: null,
+        editHtml: false,
+      );
+      testUnlocalizedBody(
+        expectation: 'body',
+        plaintextBody: false,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: true,
+        isEdit: true,
+        editBody: null,
+        editFormattedBody: null,
+        editHtml: true,
+      );
+      testUnlocalizedBody(
+        expectation: '**formatted body**',
+        plaintextBody: true,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: true,
+        isEdit: true,
+        editBody: null,
+        editFormattedBody: null,
+        editHtml: false,
+      );
+      testUnlocalizedBody(
+        expectation: '**formatted body**',
+        plaintextBody: true,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: true,
+        isEdit: true,
+        editBody: null,
+        editFormattedBody: null,
+        editHtml: true,
+      );
+
+      // everything where we expect the formatted body to be returned
+      testUnlocalizedBody(
+        expectation: '**formatted body**',
+        plaintextBody: true,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: true,
+        isEdit: false,
+        editBody: null,
+        editFormattedBody: null,
+        editHtml: false,
+      );
+      testUnlocalizedBody(
+        expectation: '**formatted body**',
+        plaintextBody: true,
+        body: 5,
+        formattedBody: '<b>formatted body</b>',
+        html: true,
+        isEdit: false,
+        editBody: null,
+        editFormattedBody: null,
+        editHtml: false,
+      );
+
+      // everything where we expect the edit body to be returned
+      testUnlocalizedBody(
+        expectation: 'edit body',
+        plaintextBody: false,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: false,
+        isEdit: true,
+        editBody: 'edit body',
+        editFormattedBody: null,
+        editHtml: false,
+      );
+      testUnlocalizedBody(
+        expectation: 'edit body',
+        plaintextBody: false,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: false,
+        isEdit: true,
+        editBody: 'edit body',
+        editFormattedBody: '<b>edit formatted body</b>',
+        editHtml: true,
+      );
+      testUnlocalizedBody(
+        expectation: 'edit body',
+        plaintextBody: false,
+        body: 'body',
+        formattedBody: '<b>formatted body</b>',
+        html: true,
+        isEdit: true,
+        editBody: 'edit body',
+        editFormattedBody: '<b>edit formatted body</b>',
```

Comment:
hm shouldn't this return the edited formatted body?

Reply:
No, because it has plaintextBody set to false.